### DISCUSSION TITLE
Reverse the behavior of `---` and `___`

### DISCUSSION THREAD

**Discussion by oredaze** [nvim-neorg/neorg#1606] [Q&A]
The last few days I took a look at this wonderful plugin. I think the syntax of neorg is almost perfect. Everything is very intuitive to me, everything except the way horizontal lines are done. I love these things, I put them everywhere. For me the intuitive thing is to use the minus `---`. However, neorg wants me to use the underscores `___` for that. The minus is taken, as you likely know.

Is there a way for me to swap the behavior of these two, or should I suck it up and live with this flaw? I don't want to resort to editing source code, because updating will be a pita.
So close to perfection...

**Comment by pergatore** [comment]
The formatting is actually done/decided by the format .norg [https://github.com/nvim-neorg/norg-specs](norg) and not by neorg (though the creators are the same) but the issue should be raised there, I believe.
